### PR TITLE
Feature: Display total number event cards

### LINKED ISSUE #2529: [FEATURE] Display event totals in the relevant cards
### Description

After this PR is merged https://github.com/EddieHubCommunity/LinkFree/pull/2527

Show totals for each in the relevant cards

Screenshot 2022-12-28 at 22 09 27

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ThorOnTheRocks**
Closes #2529 

## Changes proposed

Adding the total number of events on each card type event

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-01-02 at 16 56 38

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2594"></a>

### CODE REVIEW

File: components/event/EventKey.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import { IconContext } from "react-icons";
 import { MdOutlineOnlinePrediction, MdOutlinePeople } from "react-icons/md";
 
-export default function EventKey() {
+export default function EventKey({ events }) {
+  const inPersonEvents = events.filter(event => event.isInPerson)
+    .map(inPersonEvent => inPersonEvent)
```
Comment (eddiejaoude) [inline components/event/EventKey.js:6]: Is the map needed, could it just be `events.filter(event => event.isInPerson).length`?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you 👍  I think your fork's branch is out of date because it containers other changes

File: components/event/EventKey.js (hunk 1 above)
Comment (ThorOnTheRocks) [reply components/event/EventKey.js:6]: Not at all the map was pretty useless indeed. I was getting back undefined at first and I thought the map was fixing this.

**Comment by ThorOnTheRocks** [comment]
> 

@eddiejaoude I'm not sure because I did a fetch on my fork branch before pulling it on my local machine...I need to double-check! Thanks!

**Comment by eddiejaoude** [comment]
Thanks for double checking, something still doesn't seem quite right. Can you checkout the files that should have change with the `main` branch?

For example you can use

```
git checkout main -- CHANGELOG.md
```

**Pull request closed without merging**